eq-author-api/middleware/loadQuestionnaire.js

```diff
@@ -1,14 +1,14 @@
 const { getQuestionnaire } = require("../utils/datastore");
 
-module.exports = (logger, context) => async (req, res, next) => {
+module.exports = () => async (req, res, next) => {
```

Comment:
Don't think we need to pass in logger or ctx on line 38 in app.js with this change.